### PR TITLE
Add mutable filter

### PR DESCRIPTION
**Pull request by TheoBrigitte**
This PR adds the capability to check if owner can change token metadata

### CODE REVIEW

File: filters/mutable.filter.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { getMetadataAccountDataSerializer, MetadataAccountData } from '@metaplex-foundation/mpl-token-metadata';
+import { logger } from '../helpers';
+
+export class MutableFilter implements Filter {
+  constructor(private readonly connection: Connection) {}
+
+  async execute(poolKeys: LiquidityPoolKeysV4): Promise<FilterResult> {
+    try {
+      const metadataPDA = getPdaMetadataKey(poolKeys.baseMint);
+      const metadataAccount = await this.connection.getAccountInfo(new PublicKey(metadataPDA.publicKey.toString()));
```
Comment (fdundjer) [inline filters/mutable.filter.ts:14]: metadataPDA.publicKey is already of type PublicKey, no need to create new instance

File: filters/mutable.filter.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      if (!metadataAccount?.data) {
+        return { ok: false, message: 'Mutable -> Failed to fetch account data' };
+      }
+      const deserialize = getMetadataAccountDataSerializer().deserialize(metadataAccount.data);
```
Comment (fdundjer) [inline filters/mutable.filter.ts:18]: A possible optimization is to move
`getMetadataAccountDataSerializer()`
to constructor.

File: filters/mutable.filter.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { Filter, FilterResult } from './pool-filters';
+import { Connection, PublicKey } from '@solana/web3.js';
+import { LiquidityPoolKeysV4 } from '@raydium-io/raydium-sdk';
+import { getPdaMetadataKey } from '@raydium-io/raydium-sdk';
+import { getMetadataAccountDataSerializer, MetadataAccountData } from '@metaplex-foundation/mpl-token-metadata';
```
Comment (fdundjer) [inline filters/mutable.filter.ts:5]: @metaplex-foundation/mpl-token-metadata should be added to package.json

File: filters/mutable.filter.ts (hunk 1 above)
Comment (TheoBrigitte) [reply filters/mutable.filter.ts]: fixed

File: filters/mutable.filter.ts (hunk 2 above)
Comment (TheoBrigitte) [reply filters/mutable.filter.ts]: added new metadataSerializer into constructor

File: filters/mutable.filter.ts (hunk 3 above)
Comment (TheoBrigitte) [reply filters/mutable.filter.ts]: added @metaplex-foundation/mpl-token-metadata to package.json

**Comment by TheoBrigitte** [comment]
Just realized I set both mutable and freezable checks to true by default. Do we really want this ?

**Comment by fdundjer** [comment]
> Just realized I set both mutable and freezable checks to true by default in `.env.copy`. Do we really want this ?

I would set them to false by default

**Comment by TheoBrigitte** [comment]
> > Just realized I set both mutable and freezable checks to true by default in `.env.copy`. Do we really want this ?
> 
> I would set them to false by default

That's done

**Pull request merged**